week2/game_2048/class_version/merge.py

```diff
@@ -0,0 +1,26 @@
+def merge(line):
+    """
+    Function that merges a single row or column in 2048.
+    """
+    non_zeros = try_merge([value for value in line if value != 0])
+    zeros = [0 for dummy_num in range(len(line) - len(non_zeros))]
+    return (non_zeros + zeros)
+
+def try_merge(line_wo_zeros):
+    """
+    Attempts to merge each number in an array with the one after it.
+    If the number was already merged with the one before it, 
+    no merge will occur.
+    """
+    return [ get_new_value(line_wo_zeros, index, value) for index, value in enumerate(line_wo_zeros)]
```

Comment:
For best practices, I recommend running a linter against your code. It would have picked up the space after the `[`.

I also recommend you install those testing and coverage tools to report on that, too.